posthog/api/element.py

```diff
@@ -119,15 +128,28 @@ def stats(self, request: request.Request, **kwargs) -> response.Response:
                 )
 
             with timer("serialize_elements"):
-                serialized_elements = [
-                    {
+                distinct_chains = set()
```

Comment:
count how many distinct chains we have

and separately time the processing and the serializing to json